<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamsDefinedEditor</class>
 <widget class="QWidget" name="ParamsDefinedEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddParams">
       <property name="text">
        <string>新增参数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteParams">
       <property name="text">
        <string>删除参数</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableParams">
     <property name="columnCount">
      <number>6</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnAddParams</tabstop>
  <tabstop>btnDeleteParams</tabstop>
  <tabstop>tableParams</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
